### ISSUE TITLE
Transformer en serveur

### ISSUE THREAD

**Issue by crubier** [FFCK/pce-utils#1]
Avec un petit effort, il serait possible de transformer ces scripts en un serveur web simple en express ou koa, avec deux endpoints:

  - `/convert` qui reçoit des requetes avec le contenu du fichier siwidata, et qui renvoie une réponse avec le contenu du PCE associé (=faire tourner le script, simplement) (@niabb tu peux t'en occuper ?)
  - `/` qui servirait une petite page html. La page html permetrait d'uploader un fichier siwiData, appelle le endpoint `/convert` , et permet de télécharger le résultat (je peux me charger de cette partie)

Ensuite on pourrait déployer ce service pour le rendre accessible à tous depuis n'importe ou, sans installation

**Comment by niabb** [comment]
Effectivement, le but est de permettre en plus de l'utilisation en ligne de commande, l'intégration à un service web ou à une application "Desktop" (avec Electron par exemple).

Pour la partie Siwidata, à voir l'utilité réelle en fonction du nombre de course concerné.
Par contre, étant donné le retard que prend Compet'FFCK et la complexité de certaines manips dans FFCanoe par rapport aux formats de course qui ont évolué en slalom (format inter avec Q1, Q2, SF et F sur les finales N2/N3 par exemple), il peut être intéressant de proposer aux organisateurs un outil qui simplifie la manipulation, y compris "hors-ligne", d'où l'idée d'une petite appli "Desktop".

Dans tous les cas, il faut dans un premier temps que je restructure un peu le code, j'ai fait les développement dans l'urgence "à l'arrache".

Mais sinon d'accord avec toi, c'est une bonne idée !
